app/assets/javascripts/repositories/index.js

```diff
@@ -104,7 +104,8 @@
         }],
         fnInitComplete: function(e) {
           initActionToolbar();
-          actionToolbarComponent.setBottomOffset(68);
+          window.actionToolbarComponent.setReloadCallback(initRepositoriesDataTable, ['#repositoriesList', archived]);
```

Comment:
Instead of introducing a new parameter for this function, you can just pass a function that will call the function including the needed parameters like so:
```
window.actionToolbarComponent.setReloadCallback(() => initRepositoriesDataTable('#repositoriesList', archived));
```